qa/TL0_FW_iterators/test_jax.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages='${python_test_runner_package} numpy jax'
```

Comment:
I would add jax to setup_packages to make sure we pin the version - a not to make sure we won't forget about this.

Reply:
I am working on this. I wanted to run something in the meantime to see what happens.

The issue is I am not how to implement [JAX instalation guide for local CUDA](https://github.com/google/jax#pip-installation-gpu-cuda-installed-locally-harder) in terms of what we have in `setup_packages.py` at the moment.

Is this the right way to do it:
```
CudaPackageExtraIndex("jax[cuda11_local]",
                        {"118": [PckgVer("0.4.10", alias="jax")]},
                        extra_index="https://storage.googleapis.com/jax-releases/jax_cuda_releases.html"),
CudaPackageExtraIndex("jax[cuda12_local]",
                        {"121": [PckgVer("0.4.10", alias="jax")]},
                        extra_index="https://storage.googleapis.com/jax-releases/jax_cuda_releases.html"),
```

Reply:
How about:
```
CudaPackageExtraIndex("jax[cuda{cuda_v}[0:2]_local]",
                        {"118": [PckgVer("0.4.10", alias="jax")],
                          "121": [PckgVer("0.4.10", alias="jax")]}
                        extra_index="https://storage.googleapis.com/jax-releases/jax_cuda_releases.html"),
```?

Reply:
Cool, thanks.